buildkite/bazelci.py

```diff
@@ -2254,7 +2254,9 @@ def set_env_var(config_key, env_var_name):
     if not (
         is_pull_request()
         or use_but
-        or os.getenv("BUILDKITE_BRANCH") != "master"
+        or (os.getenv("BUILDKITE_BRANCH") != "master" 
```

Comment:
you can merge all three conditions:
... or os.getenv("BUILDKITE_BRANCH) not in ("master", "main", "stable") or ...

Reply:
For clarity we should move it out of the first "not" clause, too.